README.md

```diff
@@ -12,6 +12,7 @@ This Puppet module provides secure configuration of your base OS with hardening.
 
 * Puppet OpenSource or Enterprise
 * [Module stdlib](https://forge.puppet.com/puppetlabs/stdlib)
+* [Module sysctl](https://forge.puppet.com/thias/sysctl)
```

Comment:
This should not be part of this PR.

Reply:
yes, please rebase

Reply:
Yes of course, sorry i'm new with git stuff, will try to fix this out.